vertical-pod-autoscaler/README.md

```diff
@@ -214,6 +218,18 @@ kubectl delete clusterrolebinding myname-cluster-admin-binding
 
 # Known limitations
 
+## Limits control
```

Comment:
I don't think this section belongs in Known limitations.

Reply:
Moved.